pkg/app/appcommon/config.go

```diff
@@ -2,9 +2,10 @@ package appcommon
 
 // Config defines configuration parameters for `Proc`.
 type Config struct {
-	Name         string `json:"name"`
-	SockFilePath string `json:"sock_file_path"`
-	VisorPK      string `json:"visor_pk"`
-	BinaryDir    string `json:"binary_dir"`
-	WorkDir      string `json:"work_dir"`
+	Name       string `json:"name"`
+	ServerHost string `json:"server_host"`
+	ServerPort uint   `json:"server_port"`
```

Comment:
Should we just have `ServerAddr string`? I feel like it will make things simpler.

Reply:
Might be, agreed, will do

Reply:
Done